### ISSUE TITLE
Play music on slideshow

### ISSUE THREAD

**Issue by xmha97** [deckerst/aves#438]
Hi,
Please add the feature below:
- Play music on slideshow
Thanks

**Comment by deckerst** [comment]
Do you want to pick some music from your device when you start a slideshow?

**Comment by xmha97** [comment]
Yes but I don't like being asked for the song every time I tap on the slideshow, I think it would be better if the song was selected in the app settings.
